docs/sidebars.js

```diff
@@ -22,7 +22,7 @@ const sidebars = {
         type: "doc",
         id: "guides/README",
       },
-      items: ["guides/guidelines", "guides/examples"],
+      items: ["guides/guidelines", "guides/examples", "guides/developers.md"],
```

Comment:
```suggestion
      items: ["guides/guidelines", "guides/examples", "guides/developers"],
```

Reply:
docusaurus will understand to add the .md itself

Reply:
Done, thank you.